src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,25 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The gateway Pod has been renamed from `submariner` to `submariner-gateway`.
+* The Helm charts now support Submariner deployments using the Operator.
+* Broker creation is now managed by the Operator (and not `subctl`).
+* Each Submariner Pod now has its own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
+* The `submariner_connections` metric is renamed to `submariner_requested_connections`.
+* For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy
+  Service Discovery via the `--components` flag.
+* Improved Service CIDR discovery for k3s deployments.
+* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
+  every Service in the cluster, whether or not it was exported.
+* Lighthouse now allows a user to set name of CoreDNS custom ConfigMap.
+* The `strongswan` cable driver was removed.
+* Configuring Globalnet on a cluster is now optional with the `--globalnet` flag.
```

Comment:
Whether to enable `globalnet` or not was a deployment choice we had even earlier as well. Can you please elaborate on this point?

Reply:
From this I believe https://github.com/submariner-io/submariner-operator/pull/972/files

Reply:
This is for join - we should be specific. Looks like it was only a choice for `deploy-broker`. I'm not clear on the use case for join tho. 

```suggestion
* Globalnet can now be specifically disabled for a cluster on join via the `--globalnet` flag.
```

Reply:
Please add this change as well.